contracts/RiskFund/RiskFund.sol

```diff
@@ -33,14 +33,10 @@ contract RiskFund is
     IRiskFund
 {
     using SafeERC20Upgradeable for IERC20Upgradeable;
-
+    address public convertibleBaseAsset;
+    address private shortfall;
```

Comment:
Make it public maybe? Would help us to verify everything is correctly configured

Reply:
fixed https://github.com/VenusProtocol/isolated-pools/pull/278/commits/72211a82a65d6c3b98663baddd84f0cd44ce8886